docs/usage/getting-started/private-packages.md

```diff
@@ -324,6 +324,18 @@ npmRegistries:
 For each known NuGet registry, Renovate searches for `hostRules` with `hostType=nuget` and matching host.
 For those found, a command similar to the following is run: `dotnet nuget add source ${registryInfo.feedUrl} --configfile ${nugetConfigFile} --username ${username} --password ${password} --store-password-in-clear-text`
 
+password should be encoded to base64
+```js
```

Comment:
```suggestion
```json
```

JS: JavaScript
JSON: Javascript Object Notation <-- The one we want to use here

Reply:
i actually wanted to be consistent with other managers in the same page which also use js instead of json